lib/ui/painting/canvas.cc

```diff
@@ -596,12 +596,17 @@ Dart_Handle Canvas::drawAtlas(Dart_Handle paint_objects,
     tonic::Int32List colors(colors_handle);
     tonic::Float32List cull_rect(cull_rect_handle);
 
+    std::vector<DlColor> dl_color(colors.num_elements());
```

Comment:
I tend to use:

```
  std::vector<Foo> things;
  things.reserve(N);
  for (all the things) {
    things.emplace_back(next thing constructor arguments);
    // or things.push_back(next thing) if it's a simple thing or we've already allocated the things
  }
```
It avoids double initialization of the memory. Reserve just allocates and doesn't initialize. Since the loop will be replacing all of the default-initialized values, it's a waste of time.

Reply:
Yeah that is a better way to do it. I think I'll let this sit for now as my expectation is that we can rip it out once the wide gamut conversions are finished.

Reply:
I think there is even a lint for this that we don't have turned on.